src/StarterGui/WesTimingGui/TimingLocalScript/PlayerBehaviourHandler.lua

```diff
@@ -24,6 +24,11 @@ function handler.HandlePlayerTouch(part: BasePart)
     elseif part.Name == "_S2" then
         _keeperService:HandleSector(2)
     elseif part.Name == "_CC" then
+        -- Checks if CC invalidates next lap
+        local nextLapInvalid = part:GetAttribute("InvalidsNextLap")
```

Comment:
Can you expand `README.lua` with an explanation on how to do this? This feature should be known from the readme.